tests/copybw.cpp

```diff
@@ -149,8 +151,12 @@ int main(int argc, char *argv[])
     gdr_mh_t mh;
     BEGIN_CHECK {
         // tokens are optional in CUDA 6.0
-        // wave out the test if GPUDirectRDMA is not enabled
-        BREAK_IF_NEQ(gdr_pin_buffer(g, d_A, size, 0, 0, &mh), 0);
+        status = gdr_pin_buffer(g, d_A, size, 0, 0, &mh);
+        if (status != 0) {
+            cerr << "error in gdr_pin_buffer with code=" << status << endl;
+            cerr << "Does the selected GPU support GPUDirect RDMA?" << endl;
```

Comment:
the second diagnostic msg is barely useful.
we could use CU_DEVICE_ATTRIBUTE_GPU_DIRECT_RDMA_SUPPORTED in CUDA 11.3 to distinguish the GDR supported cases from other cases

Reply:
just filed #189

Reply:
in practice, I'm asking to to simply print the first msg.

Reply:
Done.